### ISSUE TITLE
Confusing behaviour of serialization/deserialization of message in consumer_rabbit_middleware module

### ISSUE THREAD

**Issue by mkrzywanski** [spring-cloud-samples/spring-cloud-contract-samples#291]
**Describe the bug**
I am not sure if this is a bug but it concerns `consumer_rabbit_middleware` module. The message that is produced by `StubTrigger` is serialized twice. Therfore it results in escaped json string that is consumed by listener. In the end  we end up with instance of `Foo2` that has field `foo` with value of json (`{"foo":"example"}`).

![image](https://user-images.githubusercontent.com/15279585/127779429-ec3c6282-19f7-4e77-b9ad-cc0de39e4bb7.png)

The actual body is first created in `StubRunnerExecutor::send` method (`JsonOutput.toJson` method call). Then the resulting json string is serialized again : [here](https://github.com/spring-cloud-samples/spring-cloud-contract-samples/blob/4b8f97a463cf86c37977370d8c9af52bb2df1440/consumer_rabbit_middleware/src/test/java/com/example/ApplicationTests.java#L116) when writing the payload to bytes - we end up with escaped json.

The solution would be to convert generic payload to bytes without using `ObjectMapper` (it will be `String` in this case so simple cast could be sufficient).

If it is not desired behaviour (at least I suspect so) I could create a PR for this.

**Comment by marcingrzejszczak** [comment]
I don't think this is intended. Please feel free to file a PR against contract 3.0.x branch.
